### PR TITLE
Lending analysis

### PR DESCRIPTION
**Pull request by FangFeng-077**

### CODE REVIEW

**Comment by JoshuaGRubin** [comment]
Thanks for the PR!

It looks like you got SHAP working, Fang :-)– super step!

I'd like you to

1. Add a README.md to the scripts directory describing the contents of your notebooks... i.e. what you were playing with and what the result is?  It'll make it easer for an outsider to understand quickly what to look at.
2. Take a look at [sklearn's normalize function](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.normalize.html).  Can you replace your custom normalization code with this?  It's nice to use standard tools where possible.
3.  Also take a look at `pandas.get_dummies` for converting categoricals to one hot encoded columns.  I don't know if it's any easier than your method, but might be worth looking at.

Also:

Do you have an idea of what kind of tool or final data analysis pipeline you'd like to end up with?  It's probably a good idea to eventually move toward `.py` package for reusable parts of your analysis.  It'll also make it easier for me to provide feedback on code style :-).

Happy to approve after 1... (and 2 if it makes sense to you).

**Comment by FangFeng-077** [comment]
README is added and fixed the other problems
